i18n/en.json

```diff
@@ -816,8 +816,10 @@
   "admin.experimental.clientSideCertCheck.title": "Client-Side Certification Login Method:",
   "admin.experimental.clientSideCertEnable.desc": "Enables client-side certification for your Mattermost server. See [documentation](!https://docs.mattermost.com/deployment/certificate-based-authentication.html) to learn more.",
   "admin.experimental.clientSideCertEnable.title": "Enable Client-Side Certification:",
+  "admin.experimental.collapsedThreads.always_on": "Always On",
   "admin.experimental.collapsedThreads.default_off": "Enabled (Default Off)",
-  "admin.experimental.collapsedThreads.desc": "When enabled (default off), users must enable collapsed reply threads in Settings. When disabled, users cannot access Collapsed Reply Threads. Please review our [documentation for known issues](!https://docs.mattermost.com/messaging/organizing-conversations.html) and help provide feedback in our [community channel](!https://community-daily.mattermost.com/core/channels/folded-reply-threads).",
+  "admin.experimental.collapsedThreads.default_on": "Enabled (Default On)",
+  "admin.experimental.collapsedThreads.desc": "When enabled (default off), users have the option enable Collapsed Reply Threads in Account Settings. When enabled (default on), users see Collapsed Reply Threads by default and have the option to disable it in account settings. When always on, users are required to use Collapsed Reply Threads and cannot disable it.",
```

Comment:
Please capitalize the second: "Account Settings"

Reply:
Done.